docs/decisions/0011_multiple_event_types_per_topic.rst

```diff
@@ -0,0 +1,73 @@
+10. Multiple event types per topic
+##################################
+
+Status
+******
+
+- **Accepted** DATE TO BE ADDED BEFORE MERGING
+
+Context
+*******
+
+Following `this ADR`_ in openedx-events, all implementations of event bus must support multiple event types on the same topic. This ADR explains how we will implement this using Kafka.
+**UPDATE LINK AFTER THAT ADR MERGES**
+
+.. _this ADR: https://github.com/openedx/openedx-events/pull/217
+
+Decision
+********
+To comply with the event bus requirements, we will use the TopicRecordName strategy in the event serializers, which are responsible for registering and validating schemas against the schema registry.
```

Comment:
Can you summarize why? For example: This method seems to be the simplest solution that solves the problem. (Or whatever you think the summary would be.) You don't really call out the pros/cons of the selected or rejected alternatives, so the why doesn't really jump out at you. Consider adding pros/cons, or reorganizing to make this more clear.